src/tv2-common/content/server.ts

```diff
@@ -41,67 +42,49 @@ type VTProps = Pick<
 
 export function GetVTContentProperties(
 	config: TV2BlueprintConfig,
-	file: string,
-	seek?: number,
-	sourceDuration?: number
+	contentProps: Omit<ServerContentProps, 'mediaPlayerSession'>
 ): VTProps {
 	return literal<VTProps>({
-		fileName: file,
+		fileName: contentProps.file,
 		path: JoinAssetToNetworkPath(
 			config.studio.ClipNetworkBasePath,
 			config.studio.ClipFolder,
-			file,
+			contentProps.file,
 			config.studio.ClipFileExtension
 		), // full path on the source network storage
 		mediaFlowIds: [config.studio.ClipMediaFlowId],
-		sourceDuration: sourceDuration && sourceDuration > 0 ? sourceDuration : undefined,
+		sourceDuration:
+			contentProps.sourceDuration && contentProps.sourceDuration > 0 ? contentProps.sourceDuration : undefined,
 		postrollDuration: config.studio.ServerPostrollDuration,
 		ignoreMediaObjectStatus: config.studio.ClipIgnoreStatus,
-		seek
+		seek: contentProps.seek
 	})
 }
 
 export function MakeContentServer(
 	context: IShowStyleUserContext,
-	file: string,
-	mediaPlayerSessionId: string,
 	partDefinition: PartDefinition,
 	config: TV2BlueprintConfig,
 	sourceLayers: MakeContentServerSourceLayers,
-	adLibPix: boolean,
-	voLevels: boolean,
-	seek?: number,
-	sourceDuration?: number
+	props: ServerPartProps,
+	contentProps: ServerContentProps
 ): WithTimeline<VTContent> {
 	return literal<WithTimeline<VTContent>>({
-		...GetVTContentProperties(config, file, seek, sourceDuration),
+		...GetVTContentProperties(config, contentProps),
 		ignoreMediaObjectStatus: true,
-		timelineObjects: GetServerTimeline(
-			context,
-			file,
-			mediaPlayerSessionId,
-			partDefinition,
-			config,
-			sourceLayers,
-			adLibPix,
-			voLevels,
-			seek
-		)
+		timelineObjects: GetServerTimeline(context, partDefinition, config, sourceLayers, props, contentProps)
 	})
 }
 
 function GetServerTimeline(
 	context: IShowStyleUserContext,
-	file: string,
-	mediaPlayerSessionId: string,
 	partDefinition: PartDefinition,
 	config: TV2BlueprintConfig,
 	sourceLayers: MakeContentServerSourceLayers,
-	adLibPix?: boolean,
-	voLevels?: boolean,
-	seek?: number
+	props: ServerPartProps,
```

Comment:
`props` is a little too vague about what's actual in the variable. (I know it's commenly used for React components, which is fine, but this is not a React component)

Since there is also a `contentProps` variable which is a `ServerContentProps`, it would be nice to rename `props` to `partProps` to avoid potential confusion. (Personally I would prefer the variables to be named `serverPartProps` and `serverContentProps`, but I will let you decide if you want to take it that far)

Sidenote: I do see that there are other methods in here that uses this naming for these exact parameters, so I get if you went with it to keep consistency (which is nice)